features/step-definitions/all-steps.js

```diff
@@ -0,0 +1,148 @@
+const apiKey = require('../../models/api-key.js');
+const assert = require('assert');
+const crypto = require('crypto');
+const {defineSupportCode} = require('cucumber');
+const encryption = require('../../helpers/encryption.js');
+const password = require('../../helpers/password.js');
+
+defineSupportCode(function({Given, When, Then}) {
+  let aesKeyBase64;
+  let apiKeyRecord;
+  let apiSecret;
+  let apiSecret2;
+  let apiSecretValidationResult;
+  let decryptedMessage;
+  let encryptedMessage;
+  let hashedApiSecret;
+  let hashedApiSecretCheck;
+  let plainTextMessage;
+  
+  Given('I do not have an API Key record', function () {
```

Comment:
to be consistent, you capitalized NOT on 24...

Reply:
Thanks, fixed in 2d19752